node-packages/commons/src/tasks.ts

```diff
@@ -1148,7 +1148,15 @@ export const createMiscTask = async function(taskData: any) {
           // Handle setting up the configuration for a restic restoration task
           const restoreName = `restore-${R.slice(0, 7, taskData.data.backup.backupId)}`;
           // generate the restore CRD
-          const restoreConf = restoreConfig(restoreName, taskData.data.backup.backupId, makeSafe(taskData.data.project.name))
+
+          let baasBucketName = ""
+          for (let variable of osResult.envVariables) {
```

Comment:
The result of `getOpenShiftInfoForProject` is an object keyed by `project` so you'd want to do `osResult.project.envVariables` or use `osProjectOpenShift.envVariables`.